hkmc2/shared/src/main/scala/hkmc2/semantics/Elaborator.scala

```diff
@@ -642,9 +642,16 @@ extends Importer:
               val s = st.map(term(_)(using newCtx))
               val b = rhs.map(term(_)(using newCtx))
               val r = FlowSymbol(s"‹result of ${sym}›")
+              val isGetter = k === syntax.Fun && td.paramLists.isEmpty && b.nonEmpty
```

Comment:
Because function declarations also have empty parameter lists:
```js
fun a() = 1
// paramLists = List(Tup(List()))

fun b = 1
// paramLists = List()

fun c: Int -> Int
// paramLists = List()
```